rasa/core/processor.py

```diff
@@ -485,7 +496,7 @@ def should_predict_another_action(action_name, events) -> bool:
         for e in events:
             if isinstance(e, ReminderCancelled):
                 name_to_check = (
-                    str(e.action_name)
+                    str(e.intent)
```

Comment:
also wondering how the entities play in here

Reply:
As mentioned above, I think entities should not be compared here.

Reply:
see my response above